public/app/ui/pages/detail/DetailPage.js

```diff
@@ -37,6 +38,10 @@
                     return 'collections/' + context.params.type + '/detail';
                 });
 
+                this.curatorUrl = ko.pureComputed(function () {
+                    return providence.baseUrl.objects + 'object_id/' + context.params.id;
```

Comment:
The `object_id` is part of a key/value pair, zend framework style, so I kept it here.